src/Catty/ViewController/Download/StoreProjectDownloader.swift

```diff
@@ -183,3 +197,22 @@ enum ProjectType {
     case mostViewed
     case mostRecent
 }
+
+struct ProjectFetchFailureInfo: Equatable {
+    var type: ProjectType
+    var url: String
+    var statusCode: Int?
+    var description: String?
+
+    func dict() -> [String: Any] {
```

Comment:
Building the `userInfo` dictionary is something only important for _Crashlytics_. Please do that inside the _Crashlytics_ code and not here at the downloader directly.